keps/sig-api-machinery/20190228-priority-and-fairness.md

```diff
@@ -1782,6 +1782,19 @@ Alpha:
 - Adequate documentation for the changes
 - Minimum viable test cases mentioned in Test Plan section
 
+Beta:
+
+- Blocking Items:
+  - Improving observability and robustness: finer metrics and adding debug endpoint dumping fine-grained states of the queues for priority-levels
+  - Opt-out client-side rate-limitting to prove the feature
```

Comment:
Can you expand this line to match the verboseness of the previous line? People who didn't see the conversation here will likely not understand what this means.

Reply:
expanded the line, PTAL